modules/gitops-release-notes-1-6-0.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+[id="gitops-release-notes-1-6-0_{context}"]
+= Release notes for {gitops-title} 1.6.0
+
+{gitops-title} 1.6.0 is now available on {product-title} 4.9, 4.10 and 4.11.
+
+[id="new-features-1-6-0_{context}"]
+== New features
+
+The current release adds the following improvements:
+
+* With this update, the parameters of `spec.sso` field that controlled the keycloak configurations are moved to `.spec.sso.keycloak`.
+The parameters of `.spec.dex` field are moved to `.spec.sso.dex`. Dex can now be enabled/disabled using `.spec.sso.provider`. The existing configuration methods will be supported alongside the new configuration methods until GitOps v1.9, when support for `.spec.dex`, `DISABLE_DEX` and `.spec.sso` fields for keycloak configuration will be removed from the operator link:https://issues.redhat.com/browse/GITOPS-1983[GITOPS-1983]
+
+* With this update, the Argo CD notifications controller is available as an optional workload that can be enabled/disabled through `.spec.notifications.enabled` parameter in the Argo CD CR. link:https://issues.redhat.com/browse/GITOPS-1917[GITOPS-1917]
```

Comment:
It should be Argo CD Notifications Controller.  We should mention that Argo CD Notifications is released as Technical Preview, see the description in https://issues.redhat.com/browse/GITOPS-927